placement-rules-in-sql.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Placement Rules in SQL
+summary: Learn how to schedule placement of tables and partitions.
+---
+
+# Placement Rules in SQL
+
+> **Warning:**
+>
+> Placement Rules in SQL is an experimental feature. The syntax might change before its GA, and there might also be bugs.
+>
+> If you understand the risks, you can enable this experiment feature by executing `SET GLOBAL tidb_enable_alter_placement = 1;`.
```

Comment:
Can we add a note like:

```
We recommend using this feature with the help of the official tidb team to help solve specific scenarios. Over-configuration of the placement plan will increase the difficulty of maintenance and troubleshooting, so reasonable planning is very necessary. Usually, the points that need to be considered, including disaster recovery issues, high availability issues, performance issues and so on.
```

Reply:
Hi @nolouch. Some comments in line:

> We recommend using this feature with the help of the official tidb team to help solve specific scenarios. Over-configuration of the placement plan will increase the difficulty of maintenance and troubleshooting, so reasonable planning is very necessary.

To a large degree we removed over configuration as a possibility between the original proposal and the newer proposal. You can no longer specify multiple constraints; and syntactic sugar constraints are mutually exclusive with regular constraints. The only complex scenarios that remain are both `FOLLOWER_CONSTRAINTS` and `CONSTRAINTS` can apply, and a user could set a complex constraint in dictionary format.

>  Usually, the points that need to be considered, including disaster recovery issues, high availability issues, performance issues and so on.

We don't document this well in the current experimental version. Because it is experimental, users are not encouraged to use it in production, so high availability/DR did not seem relevant (yet).

I have an alternative suggestion:

> We value your feedback when evaluating experimental features. Please feel free to [create a GitHub issue](https://github.com/pingcap/tidb/issues/new/choose) or reach out to us on [TiDB Slack](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap). We are interested to both the problems you encounter and your intended use-cases.